client/src/quic_client.rs

```diff
@@ -65,14 +65,14 @@ impl TpuConnection for QuicTpuConnection {
         &self.client.addr
     }
 
-    fn send_wire_transaction(&self, data: Vec<u8>) -> TransportResult<()> {
```

Comment:
let's put the interface changes in separately

Reply:
Separate PR for the interface change?